### PR TITLE
send size of file from Easel to pylons app

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
Sending 3d object size in the recipe also.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -53,7 +53,7 @@ android {
     }
 
     signingConfigs {
-        debug {
+        release {
```
Comment (kjawadDeveloper2) [inline android/app/build.gradle]: Kindly dont change it.

File: lib/easel_provider.dart [hunk 2]
```diff
@@ -245,6 +247,10 @@ class EaselProvider extends ChangeNotifier {
                 StringParam(key: "NFT_Format", value: _nftFormat.format),
                 StringParam(key: "NFT_URL", value: "$ipfsDomain/${uploadResponse.data?.value?.cid ?? ""}"),
                 StringParam(key: "Creator", value: artistNameController.text.trim()),
+
+
+                //Saving the size of an asset
+                StringParam(key: "Size", value: _fileSize.trim()),
```
Comment (kjawadDeveloper2) [inline lib/easel_provider.dart]: Kindly add this string in the constant file.

**Review by kjawadDeveloper2 — COMMENTED** [review]
Kindly solve it and check it on table too.

**Comment by kjawadDeveloper2** [comment]
This  PR is ready for review.

File: android/app/build.gradle [hunk 3]
```diff
@@ -53,12 +53,12 @@ android {
     }
 
     signingConfigs {
-        debug {
-            storeFile file('keystore.jks')
-            storePassword 'tech.pylons'
-            keyAlias 'key_tech_pylon'
-            keyPassword 'tech.pylons'
-        }
+//        debug  {
```
Comment (mustafapylons) [inline android/app/build.gradle:56]: Is it necessary to keep these debug statements?

**Review by mustafapylons — COMMENTED** [review]
One small comment, otherwise code is good.

**Review by kiroj305 — APPROVED** [review]

File: android/app/build.gradle (hunk 3 above)
Comment (kjawadDeveloper2) [reply android/app/build.gradle:56]: yes sir. We need it when we release builds.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Pull request merged**